config/vufind/sitemap.ini

```diff
@@ -50,6 +50,27 @@ index[] = "Solr,/Record/"
 ;    other limiting options and requires that the index has terms enabled.
 retrievalMode = search
 
+;  This settings controls whether different language versions are added to the
+;  sitemap. Default is false.
+;  Note that including language versions will make the sitemap files substantially
+;  larger and may not be very useful if language detection is enabled in VuFind.
+indexLanguageVersions = false
+
+;  This setting can be used to filter the included language codes. By default all
```

Comment:
Is it worth adding "when indexLanguageVersions is true," or is that already implied strongly enough?